### PR TITLE
add long-read-mngs to the CLI

### PR DESCRIPTION
**Pull request by morsecodist**
Adds support for uploading long-read-mngs samples from the CLI. One thing to note is that this is a breaking change. Previously, we had a top level `short-read-mngs` command. I modified it to create a `metagenomics` command to be more in line with the web application and consensus genomes flow. Ordinarily I think this would be a big deal but this CLI is so rarely used I think it is better to ensure consistency than backwards compatibility.

I handled the switch in a similar way to consensus genomes.

### CODE REVIEW

File: cmd/metagenomics/shortReadMNGS.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package metagenomics
+
```
Comment (rzlim08) [inline cmd/metagenomics/shortReadMNGS.go:2]: Should the name of this file change as well?

File: cmd/metagenomics/shortReadMNGS.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"github.com/spf13/viper"
+)
+
+var projectName string
+var stringMetadata map[string]string
+var metadataCSVPath string
+var disableBuffer bool
+var technology string
+var guppyBasecallerSetting string
+var workflow string = "short-read-mngs"
```
Comment (rzlim08) [inline cmd/metagenomics/shortReadMNGS.go:20]: Can this get set with "long-read-mngs" ?

**Review by rzlim08 — APPROVED** [review]
Looks good, just a couple of nits that aren't blocking. Is there any documentation that we need to change?

File: cmd/metagenomics/shortReadMNGS.go (hunk 1 above)
Comment (morsecodist) [reply cmd/metagenomics/shortReadMNGS.go:2]: thanks for the callout good point

File: cmd/metagenomics/shortReadMNGS.go (hunk 2 above)
Comment (morsecodist) [reply cmd/metagenomics/shortReadMNGS.go:20]: it gets set with an if below

**Pull request merged**